team/compress.py

```diff
@@ -0,0 +1,6 @@
+"""Instances of modules used to compress transfers."""
+from flask_compress import Compress
+from flask.ext.htmlmin import HTMLMIN
+
+compress = Compress()
```

Comment:
What is this shit?  Get rid of it if possible.

Reply:
Made changes in the latest commit titled - <b>"Making changes to remove cache, compress, json and nav files for consistency"</b>